vendors/swedbank-pay-core-library/src/SwedbankPay/Core/Core.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace SwedbankPay\Core;
+
+use SwedbankPay\Api\Client\Client;
+use SwedbankPay\Core\Library\PaymentInfo;
+use SwedbankPay\Core\Library\TransactionAction;
+use SwedbankPay\Core\Library\OrderAction;
+use SwedbankPay\Core\Library\Methods\Card;
+use SwedbankPay\Core\Library\Methods\Invoice;
+use SwedbankPay\Core\Library\Methods\Swish;
+use SwedbankPay\Core\Library\Methods\Vipps;
+use SwedbankPay\Core\Library\Methods\Checkout;
+
+class Core implements CoreInterface
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class Core has a coupling between objects value of 15. Consider to reduce the number of dependencies under 13.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)